data/builtin_mount_options.conf

```diff
@@ -4,7 +4,8 @@ allow=exec,noexec,nodev,nosuid,atime,noatime,nodiratime,ro,rw,sync,dirsync,noloa
 vfat_defaults=uid=,gid=,shortname=mixed,utf8=1,showexec,flush
 vfat_allow=uid=$UID,gid=$GID,flush,utf8,shortname,umask,dmask,fmask,codepage,iocharset,usefree,showexec
 
-exfat_defaults=uid=,gid=,iocharset=utf8,namecase=0,errors=remount-ro
+# common options for both the native kernel driver and exfat-fuse
+exfat_defaults=uid=,gid=,iocharset=utf8,errors=remount-ro
 exfat_allow=uid=$UID,gid=$GID,dmask,errors,fmask,iocharset,namecase,umask
```

Comment:
Can we add the `namespace` option to the list of allowed options? I am not sure how important the option is with the fuse driver but we could at least allow using it without having to change the config.

Reply:
Interesting, I suppose this applies to any `fuse` filesystem and should be also added to `ntfs_allow`, right?

Reply:
Sorry for confusion here, I was thinking about the `namecase` option and it is in the allowed options. You can ignore this.